src/VirtoCommerce.CustomerModule.Web/Scripts/blades/member-list.tpl.html

```diff
@@ -48,7 +48,7 @@
 <script type="text/ng-template" id="member-list-icon.cell.html">
     <div class="ui-grid-cell-contents">
         <div class="product-img" ng-attr-title="{{COL_FIELD}}">
-            <i class="table-ico" ng-class="row.entity._memberTypeIcon ? row.entity._memberTypeIcon : 'fa-user'"></i>
+            <i class="table-ico" ng-class="row.entity._memberTypeIcon ? row.entity._memberTypeIcon : 'fa fa-user'"></i>
```

Comment:
not sure if this is the right place to fix blade header icon

Reply:
Its look like its proper place
![image](https://user-images.githubusercontent.com/47894313/112316477-cfe72a00-8cb3-11eb-8f55-a5bcb8aa9e39.png)
![image](https://user-images.githubusercontent.com/47894313/112316548-dfff0980-8cb3-11eb-9740-71a836bca963.png)

Reply:
Yea, already fixed here https://github.com/VirtoCommerce/vc-module-customer/releases/tag/3.22.0